tests/transports/smpp/test_smpp.py

```diff
@@ -35,11 +42,19 @@ async def server(stream):
 
 @fixture
 async def transport(nursery, amqp_connection):
+    """
+    An SMPP transciever transport, with default config except connecting to the
+    TEST_PORT
+    """
     config = SmppTransceiverTransportConfig(port=TEST_PORT)
```

Comment:
Instead of a hardcoded `TEST_PORT`, could we have `tcp_server` listen on an arbitrary port and look that up in here for the config?